examples/multigpu_advtrain/evaluator.py

```diff
@@ -0,0 +1,333 @@
+import os
```

Comment:
Add a docstring at the top of the file saying what it's for

Reply:
Added in 1e805c5.